docs/MemoryMap.md

```diff
@@ -0,0 +1,23 @@
+# Memory Map
+
+This file gives an overview of the flash layout used by the nRF52833 on the micro:bit v2.
+
+Note: there is also a BLE Service named MemoryMapService, which is used to communicate the application's structure over BLE and not the layout of the entire flash storage.
+
+# Memory layout
+
+| Address | Description |
+|---------|-------------|
+| 7F000 | bootloader_settings_page [settings page](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fcom.nordic.infocenter.sdk5.v15.0.0%2Flib_bootloader.html) |
+| 7E000 | mbr_params_page / bootloader settings backup [settings page](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fcom.nordic.infocenter.sdk5.v15.0.0%2Flib_bootloader.html) |
+| 77000 | [Bootloader](https://github.com/microbit-foundation/v2-bootloader). [1] |
+| 75000 | Flash Data Storage (BLE Bonds + ?) (defined by us, may change depending on implemenation) |
```

Comment:
I'd like to be a bit more specific about what we use this bit for, as it's defined by us. Is it the bootloader/SD that touches this. Am I right in thinking that the location is defined by us, but the purpose is for SoftDevice to use?

Reply:
Two pages is the minimum size for FDS, which is a Nordic SDK component similar to MicroBitStorage or MicroBitFileSystem. The application's Nordic BLE stack, under MicroBitBLEManager, stores bond information in FDS, which uses SoftDevice to actually write to flash. The bootloader doesn't touch FDS. CODAL currently doesn't store anything else in FDS.